src/class/cdc/cdc_device.c

```diff
@@ -128,6 +128,29 @@ void tud_cdc_n_set_wanted_char (uint8_t itf, char wanted)
   _cdcd_itf[itf].wanted_char = wanted;
 }
 
+uint32_t tud_cdc_n_abort_transfer (uint8_t itf)
```

Comment:
Since we don't return any other value than 0 it should be void.
```suggestion
void tud_cdc_n_abort_transfer (uint8_t itf)
```